core/src/main/resources/jenkins/model/Jenkins/systemInfo.jelly

```diff
@@ -31,123 +31,86 @@ THE SOFTWARE.
     <st:include page="sidepanel.jelly" />
     <l:breadcrumb title="${%System Information}"/>
     <l:main-panel>
+      <l:app-bar title="${%System Information}" />
+
       <l:hasPermission permission="${app.SYSTEM_READ}">
-        <div class="jenkins-app-bar">
-          <div class="jenkins-app-bar__content">
-            <h1>${%System Properties}</h1>
-          </div>
+        <div class="jenkins-tab-pane">
+          <h2 class="jenkins-tab-pane__title">${%System Properties}</h2>
```

Comment:
Should be a set of Jelly tags rather than forcing people to copy & paste internal classes around.